CoreUtilities/src/au/gov/asd/tac/constellation/utilities/gui/JMultiChoiceComboBoxMenu.java

```diff
@@ -224,6 +224,10 @@ public final void valueChanged(final ListSelectionEvent event) {
         });
     }
 
+    public Set<ListSelectionListener> getListeners() {
```

Comment:
are you able to make this protected (since this was only added for unit test purposes)

Reply:
Fixed